<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajuda</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tutorialBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tutoriais</string>
    </property>
   </widget>
   <widget class="QPushButton" name="niveisBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Níveis de Linguagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qualitBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Avaliação Qualitativa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quantBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Avaliação Quantitativa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sobreBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sobre o LEPÊ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ensinoBt">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tela de Ensino</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
